android/capacitor/src/main/java/com/getcapacitor/JSInjector.java

```diff
@@ -69,12 +69,12 @@ public String getScriptString() {
      * @return
      */
     public InputStream getInjectedStream(InputStream responseStream) {
-        String js = "<script type=\"text/javascript\">" + getScriptString() + "</script>";
+        String js = "<script type=\"text/javascript\">" + getScriptString().replace("${", "\\${") + "</script>";
```

Comment:
Is this really needed? the second param of `replaceFirst` seems to be a string, not another regular expression, so I don't think this needs escaping